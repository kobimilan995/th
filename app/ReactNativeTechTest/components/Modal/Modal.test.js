/* eslint-env jest */
import React from 'react';
import {Text} from 'react-native';
import renderer from 'react-test-renderer';

import Modal from './Modal';

describe('Components - Modal', () => {
  it('renders - modal', () => {
    const tree = renderer
      .create(
        <Modal>
          <Text>Dummy text</Text>
        </Modal>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
