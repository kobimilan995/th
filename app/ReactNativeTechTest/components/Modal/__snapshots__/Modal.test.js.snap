// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Modal renders - modal 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "marginTop": 22,
    }
  }
>
  <Modal
    animationType="slide"
    hardwareAccelerated={false}
    transparent={true}
    visible={true}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
          "marginTop": 22,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "white",
            "borderRadius": 20,
            "elevation": 5,
            "margin": 20,
            "padding": 35,
            "shadowColor": "#000",
            "shadowOffset": Object {
              "height": 2,
              "width": 0,
            },
            "shadowOpacity": 0.25,
            "shadowRadius": 4,
          }
        }
      >
        <Text>
          Dummy text
        </Text>
      </View>
    </View>
  </Modal>
</View>
`;
